packages/web/src/common/store/cache/collections/createPlaylistSaga.ts

```diff
@@ -76,7 +85,8 @@ function* createPlaylistWorker(
     userId,
     playlist,
```

Comment:
might be good to rename this to collection too? its chill to leave playlist if you think, but maybe the const above is collection?